### ISSUE TITLE
moment.duration days in month

### ISSUE THREAD

**Issue by grimurd** [moment/moment#3179]

**Comment by icambron** [comment]
I agree that converting seconds to months and back should be symmetrical.

**Comment by pedrosimao** [comment]
No news on that? Is it safe to store a very long duration in seconds format? Like more than 5000 days, for example?
